src/components/comment/_comment_event.scss

```diff
@@ -2,74 +2,56 @@
 
 .euiCommentEvent {
   overflow: hidden;
+}
 
-  .euiCommentEvent__header {
-    @include euiBreakpoint('xs', 's') {
-      margin: -($euiSizeXS);
-
-      > div {
-        margin: $euiSizeXS;
-      }
-
-      .euiCommentEvent__headerTimestamp {
-        padding-top: $euiSizeXS * .5;
-      }
-    }
-
-    align-items: center;
-
-    .euiCommentEvent__headerData > div {
-      padding-right: $euiSizeXS;
-    }
-  }
+.euiCommentEvent__header {
+  line-height: $euiLineHeight;
+  display: flex;
+  align-items: center;
+}
 
-  .euiCommentEvent__headerData {
-    align-items: center;
-    display: flex;
-    flex-wrap: wrap;
-  }
+.euiCommentEvent__headerData {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
 
-  .euiCommentEvent__headerUsername {
-    font-weight: $euiFontWeightSemiBold;
+  > div {
+    padding-right: $euiSizeXS;
   }
+}
 
+.euiCommentEvent__headerUsername {
+  font-weight: $euiFontWeightSemiBold;
 }
 
 .euiCommentEvent--regular {
   border: $euiBorderThin;
 
   .euiCommentEvent__header {
-    @include euiBreakpoint('xs', 's') {
-      padding: $euiSizeS;
+    min-height: $euiSizeXXL;
+    background-color: $euiColorLightestShade;
+    border-bottom: $euiBorderThin;
+    padding: $euiSizeXS $euiSizeS;
 
-      /**
-       * Fix for IE when using align-items:center in an item that has min-height
-            (https://github.com/philipwalton/flexbugs/issues/231#issuecomment-362790042)
-       */
+    /**
+     * Fix for IE when using align-items:center in an item that has min-height
+        (https://github.com/philipwalton/flexbugs/issues/231#issuecomment-362790042)
+     */
+    // sass-lint:disable-block mixins-before-declarations
+    @include internetExplorerOnly {
       &::after {
         content: '';
-        min-height: inherit;
+        // Calculates the minimum height based on full header's min-height minus the vertical padding
+        min-height: $euiSizeXXL - $euiSizeS;
         font-size: 0;
         display: block;
       }
     }
+  }
 
-    /**
-      * Fix for IE when using align-items:center in an item that has min-height
-          (https://github.com/philipwalton/flexbugs/issues/231#issue-245848315)
-      */
-    @include euiBreakpoint('m', 'l', 'xl') {
-      height: $euiSize;
-    }
-
-    min-height: $euiSizeXXL;
-    background-color: $euiColorLightestShade;
-    border-bottom: $euiBorderThin;
-    padding: 0 $euiSizeS;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-
+  .euiCommentEvent__headerData {
+    // Push the actions far right
+    flex-grow: 1;
```

Comment:
This replaces what you were originally doing with `justify-content: space-between` to push the actions to the right. This fixes the the fix for IE 😆 ... Since when the fix was in it was creating a third child of the flex group, so it was putting space between the actions and the pseudo element, causing the actions to become more centered.